### PR TITLE
Adds Emergency Surgery Kit 

### PR DESCRIPTION
**Pull request by wesoda25**
## About The Pull Request

Doctors need more tools since we are putting so much focus on surgery now. I figured its not a bad idea and orange man expressed his interest so here is PR. (as an aside the name "field surgery" reeks of malpractice but mr cobby has chosen this path for us) 

## Why It's Good For The Game

above
## Changelog
:cl:
add: Adds the Emergency Surgery Kit. Contains all tools necessary for tend wounds. Medical Doctors and the CMO spawn with these, as well as extras in med vendors.
imageadd: Surgery Kit overlay. 
/:cl:

### CODE REVIEW

**Comment by wesoda25** [comment]
idk why my editor does weird shit with spacing thats not how I did it but w/e

**Comment by Tlaltecuhtli** [comment]
medlathe prints surgery tools roundstart

**Comment by FantasticFwoosh** [comment]
Feels like powercreep to give each individual doctor a scalpel looking at how each doctor has [their job contents populated with this box](https://github.com/tgstation/tgstation/pull/46282/files#diff-923a2fc64f1ea97d87e87c62728e2ecdR37), I think supplying inferior subtypes of the tools would be a better choice ([disposable scalpels](https://www.google.com/search?q=disposable+scalpel&client=firefox-b-d&tbm=isch&source=lnms&sa=X&ved=0ahUKEwiZrKrEl6_kAhXQRMAKHUoACxoQ_AUI1QEoAQ&biw=1366&bih=632#imgrc=sAHurEG04r_8pM:), [cheap hemostats](https://www.google.com/search?client=firefox-b-d&biw=1366&bih=632&tbm=isch&sa=1&ei=Wn5rXfS9FfHF8gLfpouoBQ&q=disposable+hemostat&oq=disposable+hemostat&gs_l=img.3..0j0i24.380598.382418..382791...1.0..1.317.924.10j3-1......0....1..gws-wiz-img.......0i67j0i8i30.Yr4Xg2MYSEg&ved=0ahUKEwj00vjFl6_kAhXxolwKHV_TAlUQ4dUDCAY&uact=5#imgrc=k3M0l-xDP-R1QM:) both made partially out of plastic)

**Comment by dr-bee** [comment]
needs a cautery to reseal the incision.

**Comment by XDTM** [comment]
I'd also consider giving everything but the drill/saw, to allow for more on-the-spot surgery types. I think cavity implants would be more used if it didnt require you to drag the patient/victim back to surgery to do it.

**Comment by Dorsisdwarf** [comment]
If you want to do all the on the spot surgeries you can probbably just acquire your own set of tools, especially since the major stations now all have a second set in surgery B

**Comment by Mickyan** [comment]
This seems kind of pointless unless you are going to actually give these to MDs on spawn

**Comment by optimumtact** [comment]
You could give these to every MD in their backpacks yeah

**Comment by wesoda25** [comment]
Thanks for the suggestions, may or may not give CMO a complete set of surgical tools in his belt.

**Comment by CRITAWAKETS** [comment]
I recommend making this look less medkit-y and more briefcase-ey

**Comment by LoGIRz** [comment]
CMO already has the compact defib in his belt slot so if you're thinking of giving CMO a surgery set roundstart either pack that in a specialized container that goes in a pocket/backpack sort of thing or perhaps let the CMO autosurgeon accept the surgery set implant instead of the HUD and only that as a replacement?

**Comment by anconfuzedrock** [comment]
if this kit is a bag or something, maybe it should fit specifically on the labcoat?

**Comment by ATH1909** [comment]
> CMO already has the compact defib in his belt slot so if you're thinking of giving CMO a surgery set roundstart either pack that in a specialized container that goes in a pocket/backpack sort of thing or perhaps let the CMO autosurgeon accept the surgery set implant instead of the HUD and only that as a replacement?

Maybe turn it into a beacon, where you can choose between autosurgeons that have a medHUD implant, a surgery tool implant, or maybe a third thing too?

File: code/game/objects/items/storage/boxes.dm [hunk 1]
```diff
@@ -284,6 +284,18 @@
...
 	generate_items_inside(items_inside,src)
 
+/obj/item/storage/box/surgery
+  name = "Emergency Surgery Kit"
+  desc = "A box containing the bare necessities for healing a patients wounds."
+  illustration = "surgery"
+
+/obj/item/storage/box/surgery/PopulateContents()
+  new /obj/item/surgical_drapes(src)
+  new /obj/item/reagent_containers/medigel/sterilizine(src)
```
Comment (ExcessiveUseOfCobblestone) [inline code/game/objects/items/storage/boxes.dm]: remove sterilizine, it shouldn't be common especially when you're giving them the tools that give perfect probability.

**Comment by Shapsy** [comment]
wtf

**Pull request closed without merging**